### PR TITLE
upgrade SAI to v1.4 for 201811

### PR DESCRIPTION
**Pull request by lguohan**
**- What I did**
- ~need upload mellanox sdk and sai~

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/broadcom/libsaithrift-dev.mk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # libsaithrift-dev package
 
-SAI_VER = 1.3.5
+SAI_VER = 0.9.4
```
Comment (qiluo-msft) [inline platform/broadcom/libsaithrift-dev.mk]: Version downgrade. Is it a mistake?

Comment (lguohan) [reply platform/broadcom/libsaithrift-dev.mk]: it is intended.

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest broadcom please

**Comment by lguohan** [comment]
retest broadcom please

**Comment by lguohan** [comment]
retest broadcom please

**Comment by lguohan** [comment]
retest broadcom please

**Comment by lguohan** [comment]
retest mellanox please

**Comment by stcheng** [comment]
could you resolve the conflicts?

**Comment by lguohan** [comment]
resolved.

**Pull request merged**